index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fernando's Portfolio Project</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500&family=Roboto&display=swap"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"/>
+</head>
+<body>
+    <header>
+        <a href="#" class="my-logo">Fernando´s logo</a>
+        <nav class="nav-mobile-version">
+           <i class="fa fa-bars bars-icons">
+           </i>
+        </nav>
+    </header>
+    
+    <section class="content-area">
+        <div class="header-text">
+            <h1 class="portfolio-owner">Fernando Silva</h1>
+            <h2 class="pf-owner-title">Web Software Developer</h2>
+            <p class="descriptive-paragraph">
+                I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+    </section>
```

Comment:
- [ ] Good job so far with the project. However, your project is not complete as seen from your chosen template on Figma. Please consider completing your project by adding all the contents that make up the headline section of your chosen template.
- [ ] Please do not forget to add the background shadow or image as seen in the template below.

Your toolbar and headline section:
<img width="217" alt="Loom_ITsHTwfjcX" src="https://user-images.githubusercontent.com/81468675/148627064-25e36b22-73e8-4ac6-b4de-e55b48631b78.png">

Figma toolbar and headline section:
<img width="131" alt="Loom_nAadAaJI3Q" src="https://user-images.githubusercontent.com/81468675/148626877-91deb07a-2fd0-4dca-bb82-956991b90400.png">